### ISSUE TITLE
WebAssembly support

### ISSUE THREAD

**Issue by maciej** [RoaringBitmap/roaring#194]
Go 1.11 [will introduce](https://tip.golang.org/doc/go1.11#wasm) experimental [WebAssembly](https://webassembly.org) support.
It might be a good idea to check if roaring can be compiled with WASM as target.

**Comment by lemire** [comment]
Indeed, and it could be compared with https://github.com/SalvatorePreviti/roaring-wasm

cc @SalvatorePreviti

**Comment by SalvatorePreviti** [comment]
Interesting. Didn't know this at all. The wasm should work as it is, once is compiled it should run everywhere is supported. Consider however that from my tests the WASM version is much slower than the native version, and it has no support for SSE4.2 and AVX2 of course. At the moment I am working on the NodeJS native port, I would like to go back and complete the wasm version in the future. You can already get the wasm from the npm package and try it out however.
